torch/nn/utils/_named_member_accessor.py

```diff
@@ -128,6 +151,15 @@ def get_submodule(self, name: str) -> "torch.nn.Module":
             self.memo[name] = submodule
             return submodule
 
+    def swap_submodule(self, name: str, value: "torch.nn.Module") -> "torch.nn.Module":
```

Comment:
nit: Do we want to differentiate the argument name `name` here from the `name` above in the top-level `swap_submodule`? Here, the `name` can be like an FQN (with `'.'`), whereas above, it must be only the unprefixed name.

Maybe call this one `path` if you already refer to it as "path" in the comment?

Reply:
Ah, I see that this pattern is consistent with the rest of the file.

Reply:
good point, let me update